src/StatusQ/Controls/StatusInput.qml

```diff
@@ -68,6 +74,25 @@ Item {
         anchors.rightMargin: 16
 
         maximumLength: root.charLimit
+
+        onTextChanged: {
+            if (root.validators.length) {
+                for (let idx in root.validators) {
+                    let result = root.validators[idx].validate(statusBaseInput.text)
+
+                    if (typeof result === "boolean" && result) {
+                        statusBaseInput.valid = true
+                    } else {
+                        if (!root.errors) {
+                            root.errors = {}
+                        }
+                        root.errors[root.validators[idx].name] = result
+                        statusBaseInput.valid = false
+                        break;
```

Comment:
If I understand correctly, it means that as soon as we encounter an error, we stop running subsequent validator?

I think we should let the decision to render all error or only the first one to the Error component (if we built any). But as an input I think he should still run all validators.

What do you think?

Reply:
Ah good catch! We shouldn't break there. Will update this!